### PR TITLE
Task3

### PR DESCRIPTION
**Pull request by Prateek0129**

### CODE REVIEW

File: Display-Data/src/app/app.component.ts [hunk 1]
```diff
@@ -1,21 +1,40 @@
 import { Component } from '@angular/core';
+import { Directive } from '@angular/core/src/metadata/directives';
```
Comment (upexcel) [inline Display-Data/src/app/app.component.ts]: why import directive here ?

File: Display-Data/src/app/app.component.ts [hunk 2]
```diff
@@ -1,21 +1,40 @@
...
+  id:number = 4;
+ addTask(newTask:string) {
+    let data = {'id':this.id,'task':newTask};
+    this.tasks.push(data);
+    console.log(data);
+    this.id++;
+ }
 }
 /* R&D
 export class AppComponent {
```
Comment (upexcel) [inline Display-Data/src/app/app.component.ts]: remove extra codes from pulls

File: Display-Data/src/app/app.component.ts [hunk 3]
```diff
@@ -1,21 +1,40 @@
...
+ tasks:Array<object> | null = [{'id':0,'task':"Get up"},{'id':1,'task':"Brush your teeth"},{'id':2,'task':"Have Breakfast"}];
+  
+ onSelect(todo:string) {
+    this.todo=todo;
+}
+  id:number = 4;
+ addTask(newTask:string) {
+    let data = {'id':this.id,'task':newTask};
+    this.tasks.push(data);
+    console.log(data);
```
Comment (upexcel) [inline Display-Data/src/app/app.component.ts]: remove console.logs

File: Display-Data/src/app/app.component.ts (hunk 2 above)
Comment (Prateek0129) [reply Display-Data/src/app/app.component.ts]: yes sir it was for id.

File: Display-Data/src/app/app.component.ts (hunk 3 above)
Comment (Prateek0129) [reply Display-Data/src/app/app.component.ts]: i used to debug a error in code

File: Display-Data/src/app/app.component.ts (hunk 1 above)
Comment (Prateek0129) [reply Display-Data/src/app/app.component.ts]: sir. i actually forgot about this. maybe some R&D.

**Pull request merged**